docs/v3/how-to-guides/automations/chaining-deployments-with-events.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Chaining deployments with events
```

Comment:
I started titles on the pages I worked on with "How to". Not sure if we want to continue that, but thought I'd mention in case we want to keep that consistent.
```suggestion
title: How to chain workflow runs with events
```